x-pack/plugins/index_lifecycle_management/common/types/policies.ts

```diff
@@ -100,6 +103,9 @@ export interface AllocateAction {
   require?: {
     [attribute: string]: string;
   };
+  migrate?: {
+    enabled: false;
```

Comment:
Was setting `false` here intentional?

Reply:
Yes it was, :). The default will be for enabled to be true so the only reason to ever specify this at the moment is to turn migration off. I will add a comment to that effect :+1: